cmd/soroban-cli/src/commands/txn/mod.rs

```diff
@@ -0,0 +1,58 @@
+use clap::Parser;
+
+pub mod build;
+pub mod send;
+pub mod sign;
+pub mod simulate;
+pub mod ls;
+pub mod rm;
+
+#[derive(Debug, Parser)]
+pub enum Cmd {
+    /// Simulate a transaction
+    Simulate(simulate::Cmd),
+    /// Given an identity return its address (public key)
+    Sign(sign::Cmd),
+    /// Submit a transaction to the network
+    Send(send::Cmd),
+    /// Build a transaction
+    Build(generate::Cmd),
+    /// List cached transactions
+    Ls(ls::Cmd),
+    /// Remove cached transaction(s)
+    Rm(rm::Cmd),
```

Comment:
Could we build out the simulate, sign, and send first without build and other logic? They're a higher priority imo based on https://github.com/stellar/soroban-cli/issues/1180#issuecomment-2008652119.